apps/vanilla/src/pages/blackJack/index.ts

```diff
@@ -0,0 +1,59 @@
+import { DiceGenerator } from './models/diceGenerator';
+import { TurnGenerator } from './models/turnGenerator';
+import { Player } from './models/player';
+import { AllResultsRollDice } from './models/allResultsRollDice';
+
+const buttonRollTheDice = document.querySelector<HTMLButtonElement>('.game__roll-the-dice');
+const containerPlayer1 = document.querySelector<HTMLElement>('#Player1');
+const containerPlayer2 = document.querySelector<HTMLElement>('#Player2');
+const containerAll = document.querySelector<HTMLElement>('#allResults');
```

Comment:
Please, use consistent naming pattern for all ids.
```suggestion
const containerPlayer1 = document.querySelector<HTMLElement>('#player1');
const containerPlayer2 = document.querySelector<HTMLElement>('#player2');
const containerAll = document.querySelector<HTMLElement>('#allResults');
```